stac_fastapi/extensions/stac_fastapi/extensions/core/collection_search/collection_search.py

```diff
@@ -26,11 +26,6 @@ class ConformanceClasses(str, Enum):
 
     COLLECTIONSEARCH = "https://api.stacspec.org/v1.0.0-rc.1/collection-search"
     BASIS = "http://www.opengis.net/spec/ogcapi-common-2/1.0/conf/simple-query"
-    FREETEXT = "https://api.stacspec.org/v1.0.0-rc.1/collection-search#free-text"
-    FILTER = "https://api.stacspec.org/v1.0.0-rc.1/collection-search#filter"
-    QUERY = "https://api.stacspec.org/v1.0.0-rc.1/collection-search#query"
-    SORT = "https://api.stacspec.org/v1.0.0-rc.1/collection-search#sort"
-    FIELDS = "https://api.stacspec.org/v1.0.0-rc.1/collection-search#fields"
```

Comment:
Those have been moved to each extensions Enums (will update the changelog to mention it)